tests/integration/tui/screenshots/test_task_states.filter-not-waiting-or-expired.html

```diff
@@ -8,13 +8,13 @@
 <span style="color:#000000;background:#e5e5e5">            </span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5">̿● a                                                               </span>
 <span style="color:#000000;background:#e5e5e5">         </span><span style="color:#000000;background:#e5e5e5">-</span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5">̿⊗ Y                                                                  </span>
 <span style="color:#000000;background:#e5e5e5">            </span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5">̿⊗ b                                                               </span>
-<span style="color:#000000;background:#e5e5e5">      </span><span style="color:#000000;background:#e5e5e5">-</span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5">̿⊘ 2                                                                     </span>
+<span style="color:#000000;background:#e5e5e5">      </span><span style="color:#000000;background:#e5e5e5">-</span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5">̎⊘ 2                                                                     </span>
```

Comment:
The reason for the change in icon is that previously, the held state was not updated in the data store by the test fixture: https://github.com/cylc/cylc-flow/blob/478b0d0a77efff25d27fa9ac621281df25921d22/tests/integration/tui/test_app.py#L34-L44